terrain_test.py

```
@@ -0,0 +1,86 @@
+            zone_height = terrain.heights[start]
+            stdscr.addstr(y_pos, 0, f"Zone {i+1}: x={start}-{end} (width={zone_width}), height={zone_height}")
+            y_pos += 1
+        
+        # Refresh screen
+        stdscr.refresh()
+        
+        # Handle input
+        key = stdscr.getch()
+        if key == ord('q'):
+            break
+        elif key == ord('r'):
```

Ruff (0.8.2)

Unnecessary `elif` after `break` statement

Remove unnecessary `elif`

(RET508)